app/views/layouts/main-site.html.erb

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%= render partial: 'includes/head' %>
+  </head>
+
+  <body>
+    <nav>
+      <%= render partial: 'includes/menu' %>
+    </nav>
+
+    <main>
+      <%= yield %>
+    </main>
+
+    <footer id="footer" class="container">
+      <%= render partial: 'includes/footer' %>
```

Comment:
For common/shared templates like those in `includes/`, consider moving them to other directories to make use of template inheritance:

https://guides.rubyonrails.org/layouts_and_rendering.html#template-inheritance

For example, in both the `app/views/layouts/main-site.html.erb` and `app/views/layouts/admin/admin-area.html.erb` layouts, the footer could look like this:

```erb
<footer id="footer" class="container">
  <%= render 'footer' %>
</footer>
```

And then the custom footer templates could be put in:
- `app/views/application/_footer.html.erb` for the main-site layout
- `app/views/admin/_footer.html.erb` for the admin layout


This would require moving the admin layout to `app/views/layouts/admin/admin-area.html.erb`.